autoarray/plot/mat_wrap/wrap/wrap_base.py

```diff
@@ -0,0 +1,946 @@
+
+        - plt.yticks: https://matplotlib.org/3.3.1/api/_as_gen/matplotlib.pyplot.yticks.html
+        - plt.xticks: https://matplotlib.org/3.3.1/api/_as_gen/matplotlib.pyplot.xticks.html
+
+        Parameters
+        ----------
+        manual_values : [float]
+            Manually override the tick labels to display the labels as the input list of floats.
+        """
+        super().__init__(kwargs=kwargs)
+
+        self.manual_values = manual_values
+
+    def tick_values_from(
+        self, min_value: float, max_value: float, use_defaults: bool = False
+    ) -> np.ndarray:
+        """
+        Calculate the ticks used for the yticks or xticks from input values of the minimum and maximum coordinate
+        values of the y and x axis.
+
+        Certain figures may display better using the default ticks. This method has the option `use_defaults` to return
+        None such that the defaults are used.
+
+        Parameters
+        ----------
+        min_value : float
+            the minimum value of the ticks that figure is plotted using.
+        max_value : float
+            the maximum value of the ticks that figure is plotted using.
+        use_defaults : bool
+            If `True`, the function does not return tick_values such that a plotter uses the default ticks.
+        """
+        if not use_defaults:
+            if self.manual_values is not None:
+                return np.linspace(min_value, max_value, len(self.manual_values))
+            else:
+                return np.linspace(min_value, max_value, 5)
+
+    def tick_values_in_units_from(
+        self,
+        array: arrays.Array,
+        min_value: float,
+        max_value: float,
+        units: Units,
+        use_defaults: bool = False,
+    ) -> typing.Optional[np.ndarray]:
+        """
+        Calculate the labels used for the yticks or xticks from input values of the minimum and maximum coordinate
+        values of the y and x axis.
+
+        The values are converted to the `Units` of the figure, via its conversion factor or using data properties.
+
+        Certain figures may display better using the default ticks. This method has the option `use_defaults` to return
+        None such that the defaults are used.
+
+        Parameters
+        ----------
+        array : arrays.Array
+            The array of data that is to be plotted, whose 2D shape is used to determine the tick values in units of
+            pixels if this is the units specified by `units`.
+        min_value : float
+            the minimum value of the ticks that figure is plotted using.
+        max_value : float
+            the maximum value of the ticks that figure is plotted using.
+        units : Units
+            The units the tick values are plotted using.
+        use_defaults : bool
+            If `True`, the function does not return tick_values such that a plotter uses the default ticks.
+        """
+        if use_defaults:

```

Comment:
Why would I even bother calling the function if I was going to set use_defaults to True? Just don't call the function if it is True and remove it as an argument

Reply:
Good shout.